main.go

```diff
@@ -81,6 +81,7 @@ var triggerToken = flag.String("token", "", "HTTP trigger token")
 var privateToken = flag.String("private-token", "", "User PRIVATE-TOKEN with privileges to create Build triggers")
 var gitlabURL = flag.String("url", "", "GitLab instance address")
 var shouldTriggerMerged = flag.Bool("trigger-merged", false, "Should trigger merged requests which was just merged")
+var removeSourceBranchExceptions := [...]string{"develop"}
```

Comment:
This `removeSourceBranchExceptions` variable should be read from app arguments, like the other variables above.
So then you should modify the `.env` file, the `docker-compose.yml` and the `README` as well.